lib/ex_unit/lib/ex_unit/filters.ex

```diff
@@ -82,6 +82,8 @@ defmodule ExUnit.Filters do
   """
   @spec eval(t, t, map, [ExUnit.Test.t]) :: :ok | {:error, binary}
   def eval(include, exclude, tags, collection) when is_map(tags) do
+    tags = if include_skipped?(include), do: Map.delete(tags, :skip), else: tags
```

Comment:
I think this can be written as:

    include_skip? = Enum.any?(include, &has_tag(&1, %{skip: true}, collection))

Then we can change the two clauses below to:

    {:ok, msg} when is_binary(msg) and not include_skip? ->
    {:ok, true} when not include_skip? ->

What do you think?